packages/docusaurus-theme-classic/src/theme/IconEdit/index.tsx

```diff
@@ -12,21 +12,10 @@ import type {Props} from '@theme/IconEdit';
 
 import styles from './styles.module.css';
 
+import Icon from './icon.svg';
+
 function IconEdit({className, ...restProps}: Props): JSX.Element {
-  return (
-    <svg
-      fill="currentColor"
-      height="20"
-      width="20"
-      viewBox="0 0 40 40"
-      className={clsx(styles.iconEdit, className)}
-      aria-hidden="true"
-      {...restProps}>
-      <g>
-        <path d="m34.5 11.7l-3 3.1-6.3-6.3 3.1-3q0.5-0.5 1.2-0.5t1.1 0.5l3.9 3.9q0.5 0.4 0.5 1.1t-0.5 1.2z m-29.5 17.1l18.4-18.5 6.3 6.3-18.4 18.4h-6.3v-6.2z" />
-      </g>
-    </svg>
-  );
+  return <Icon className={clsx(styles.iconEdit, className)} {...restProps} />;
```

Comment:
is it ok, if i move this style to appropriate module?

i wanted to reuse this icon in https://typescript-eslint.io/play/ but i do not want custom styling on it

![image](https://user-images.githubusercontent.com/625469/144706858-5f5f0a5c-ced2-4311-8c1f-b74638791444.png)

Reply:
I would even suggest removing all default styles & moving them to where it's used. This would likely be breaking change but it would make the icons more reusable. I even wonder why we have this React component wrapper in the first place

Reply:
rn we need them for swizzling

simplified componets are going to look like this
```ts
import IconArrow from './icon.svg';

export default IconArrow;
```